src/pages/en/guides/styling.md

```diff
@@ -176,6 +176,168 @@ You can also use the `<link>` element to load a stylesheet on the page. This sho
 
+## Cascading Order
+
+Astro components will sometimes have to evaluate multiple CSS "instructions" for the same element. For example, your component might import a CSS stylesheet, include its own `<style>` tag, *and* be rendered inside a layout.
+
+When multiple CSS rules apply to the same element, browsers first use _specificity_ and then _order of appearance_ to determine which style to show.
+
+If one rule is more _specific_ than another, no matter when the CSS rule is evaluated, it will take precedence:
+
+```astro title="MyComponent.astro"
+<style>
+  h1 { color: red }
+  div > h1 {
+    color: purple
+  }
+</style>
+<div>
+  <h1>
+    This header will be purple!
+  </h1>
+</div>
+```
+
+If two rules have the same specificity, then the _order of appearance_ is evaluated, and the last rule will take precedence:
+```astro title="MyComponent.astro"
+<style>
+  h1 { color: purple }
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be red!
+  </h1>
+</div>
+```
+
+Astro CSS rules are evaluated in this order of appearance:
+
+- **`<link>` tags in the head** (lowest precedence)
+- **imported styles**
+- **scoped styles** (highest precedence)
+
+### Scoped Styles 
+
+Using [scoped styles](#scoped-styles) does not increase the _specificity_ of your CSS, but it will always come last in the _order of appearance_. They will therefore take precedence over other styles of the same specificity. For example, if you import a stylesheet with the same rule as a scoped style, the scoped style will apply:
+
+```css title="make-it-purple.css"
+h1 {
+  color: purple;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be red!
+  </h1>
+</div>
+```
+
+If you make the imported style _more specific_, it will have higher precedence over the scoped style:
+
+```css title="make-it-purple.css"
+div > h1 {
+  color: purple;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be purple!
+  </h1>
+</div>
+```
+
+### Import Order
+
+When importing multiple stylesheets in an Astro component, the CSS rules are evaluated in the order that they are imported. A higher specificity will always determine which styles to show, no matter when the CSS is evaluated. But, when conflicting styles have the same specificity, the _last one imported_ wins:
+
+```css title="make-it-purple.css"
+div > h1 {
+  color: purple;
+}
+```
+```css title="make-it-green.css"
+div > h1 {
+  color: green;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-green.css"
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be purple!
+  </h1>
+</div>
+```
+
+If a component imports CSS, that CSS will be imported at the same time, even if the component is never used:
```

Comment:
```suggestion
Importing a component also imports its CSS style rules along with it, even if the component is never used:
```
Realized this didn't actually mention the part about a component importing another component. Could have said, "If an **imported** component imports CSS..." but that starts to get mucky. Tried to rearrange to make it as succinct as possible, to cut down on repeated "component" and "import" as much as possible, but you might have something better.

Reply:
This could get tricky to understand if previously we said that CSS doesn't leak from other components. I was thinking maybe we can include some good practices to go around it. what do you think Sarah and Dan?

Reply:
We said that _scoped_ styles don't leak - maybe the best practice here is to use Scoped Styles instead?

Reply:
Oh, and to be sure to import your layout first! Adding

Reply:
Yes, this "not leaking" tripped me up at first... I almost wrote, WAIT IS THIS TRUE??
 
So to be clear, it's only the IMPORTED styles that carry over with the component, not its `<style>` tag styles?

So, we *could* try to rephrase like "imported styles are transitive" kind of thing... if a style finds its way into your component via importing (whether imported directly via a statment, or imported indirectly via a *component* import), then it's gonna end up in the list of styles that will be evaluated. And, its precedence will depend on the order of the import statement that brought it in: whether you were importing a stylesheet directly, or a component that imported its own stylesheet.

Reply:
I think "leaking" might be the best way to describe this... tell me what you think of my rephrasing and tip in the latest commit

Reply:
I like it!